src/Request.php

```diff
@@ -57,10 +57,10 @@ public function getHeaders() : array
     /**
      * {@inheritdoc}
      */
-    public function getHeader($header) : array
+    public function getHeader($name) : array
```

Comment:
Is there a specific reason why you would rename this? Just curious tho.
I am pretty sure that this might be a BC break aswell, as we already have PHP 8 support which added method calls with named arguments which will break if you change this.

But as we are implementing psalm, I'd prefer having the `@no-named-arguments` in here. Maybe we move this change to the next major even tho I don't understand why this change is here in the first place.